### PR TITLE
Use XDG_CACHE_HOME on Linux

### PR DESCRIPTION
**Pull request by xiota**
This PR allows `ialauncher` to be installed in system directories on Linux by using the user's cache directory to download and launch games.  Behavior on Windows and macOS have not been tested, but should remain unchanged.

See https://github.com/rtts/ialauncher/issues/11

### CODE REVIEW

**Comment by JaapJoris** [comment]
Thanks for improving IALauncher! It hasn't seen any changes in quite some time, but that doesn't mean the project is dead. I still use it regularly to play my favorite childhood games, or even just to watch the title screens scroll by.

However, the maintainability of this code base is greatly assisted by the minimal number of dependencies (just `pygame`), and this PR introduces another one: `pyxdg`. While it will probably not cause any issues, I'm very weary of introducing new dependencies to old projects.

A second reservation I have against this PR is that, currently, the `games` directory is completely independent of the rest of the project. I have a personal backup of this directory, including all downloaded zipfiles, in case the Internet Archive might go offline someday. Creating a distinction between the `games` and `cache` directories remove this independence and makes it more difficult to create a backup by just copying the `games` directory somewhere.

However, I admit that the current way of storing all downloaded data in the `games` module directory is a very bad practice. Earlier versions used a directory called `games` in the user's home directory, but that was changed in [this commit](https://github.com/rtts/ialauncher/commit/d798d6fe855145188f7f71f905e03f978df32401).

In conclusion, I will politely reject this PR but I encourage you to keep using your modified version of IA Launcher if it works well for you!
